backend/modules/sse/sse_consumer.py

```
@@ -1,16 +1,17 @@
 import asyncio
-import logging
 from abc import ABC, abstractmethod
+from typing import override
```

Pylint (3.3.7)

[error] No name 'override' in module 'typing'

(E0611)